editor.planx.uk/src/components/MoreInfoFeedback.tsx

```diff
@@ -27,41 +27,101 @@ const FeedbackBody = styled(Box)(({ theme }) => ({
 }));
 
 const MoreInfoFeedbackComponent: React.FC = () => {
-  return (
-    <>
+  type View = "yes/no" | "input" | "thanks";
+
+  type Sentiment = "helpful" | "unhelpful";
```

Comment:
I've also tried to reflect the language of the `feedback_type_enum` here so that it hopefully makes things easier when the forms are wired up.